ui/src/main/java/org/pentaho/mantle/client/dialogs/scheduling/ParameterPreviewSidebar.java

```diff
@@ -36,10 +36,12 @@ public class ParameterPreviewSidebar extends PromptDialogBox {
   private static final int PADDING = 20;
   private final JsJob job;
   private VerticalPanel paramsPanel;
+  private boolean hideInternalVariables;
 
-  public ParameterPreviewSidebar( JsJob job ) {
+  public ParameterPreviewSidebar( JsJob job, final boolean hideInternalVariables ) {
```

Comment:
We are adding a parameter to a constructor. Did you verify the usages of this constructor to ensure all handle passing 2 parameters instead of 1?

Reply:
Yes I have verified the entire pentaho-scheduler-plugin and I see only this place the constructor is been used which is updated with 2 parameters now.